test/test_optimize.py

```diff
@@ -627,6 +627,28 @@ def test_attrs(self):
         with self.assertRaisesRegex(AttributeError, 'Unknown variable name:'):
             dummy = fit.c
 
+    def test_pickle_ability(self):
+        """Ensure fit objects are pickle-able and copy-able"""
+
+        import pickle
+
+        csvfiles_path = os.path.join(
+            DATAFILES_PATH, 'optimize', 'rosenbrock_mle.csv'
+        )
+        fit = from_csv(path=csvfiles_path)
+        pickled = pickle.dumps(fit)
+        unpickled = pickle.loads(pickled)
```

Comment:
should we add fit delete step here (if the unpickled would reference something from fit?)